Mathlib/Data/PNat/Xgcd.lean

```diff
@@ -70,9 +70,9 @@ instance : SizeOf XgcdType :=
  reflects the matrix/vector interpretation as above. -/
 instance : Repr XgcdType where
   reprPrec
-  | g, _ => s!"[[[ {repr (g.wp + 1)}, {(repr g.x)} ], [" ++
-            s!"{repr g.y}, {repr (g.zp + 1)}]], [" ++
-            s!"{repr (g.ap + 1)}, {repr (g.bp + 1)}]]"
+  | g, _ => s!"[[[ {repr (g.wp + 1)}, {(repr g.x)} ], \
+                 [{repr g.y}, {repr (g.zp + 1)}]], \
+                [{repr (g.ap + 1)}, {repr (g.bp + 1)}]]"
```

Comment:
the spaces look a bit inconsistent here (also in the original), is this deliberate?

Reply:
I don't know what's going on with the additional spaces in the original. I got as far as figuring out that it's `[2x2 matrix, vector]` and tried lining it up in the source code.

Reply:
@digama0 Should we try to fix the whitespace for this repr instance before merging? or could we leave it as-is?

Reply:
I think unless more information is forthcoming it should be fine to just remove the spaces on the first line.

Reply:
Done

Reply:
@digama0 Do want to make any more changes, or are you happy with sending this PR onto the queue?

(Just pinging you because I'm not sure what your approval means precisely, whether that means you're waiting for another maintainer's opinion, whether it means you've delegated it to me, or whether it would be ok if I sent this to the maintainer merge queue myself.)